Tests/AsyncQueueTests/FIFOQueueTests.swift

```diff
@@ -24,34 +24,29 @@ import XCTest
 
 @testable import AsyncQueue
 
-final class AsyncQueueTests: XCTestCase {
+final class FIFOQueueTests: XCTestCase {
 
     // MARK: XCTestCase
 
     override func setUp() async throws {
         try await super.setUp()
 
-        systemUnderTest = AsyncQueue()
-    }
-
-    override func tearDown() async throws {
```

Comment:
this `tearDown` method did not have any effect on whether tests would pass or not, so I deleted it